charts/devtron/templates/NOTES.txt

```diff
@@ -7,4 +7,23 @@
 2. You can watch the progress of Devtron microservices installation by the following command
 
    kubectl -n devtroncd get installers installer-devtron -o jsonpath='{.status.sync.status}'
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+3. You can access devtron dashboard with following.
+{{- if $.Values.components.devtron.ingress.enabled }}
+
+  {{ printf $.Values.components.devtron.ingress.host }}
```

Comment:
Print the url like - Your devtron dashboard will be available on http://ingress_host/dashboard or for https, https://ingress_host/dashboard
get ingress host from values
/dashboard needs to be added manually to ingress, it doesn't redirect automatically